lib/node_modules/@stdlib/blas/base/wasm/scasum/docs/types/index.d.ts

```diff
@@ -0,0 +1,317 @@
+	* var mem = new Memory({
+	*     'initial': 10,
+	*     'maximum': 100
+	* });
+	*
+	* // Create a BLAS routine:
+	* var mod = new scasum.Module( mem );
+	* // returns <Module>
+	*
+	* // Initialize the routine:
+	* mod.initializeSync();
+	*
+	* // Define a vector data type:
+	* var dtype = 'complex64';
+	*
+	* // Specify a vector length:
+	* var N = 4;
+	*
+	* // Define pointer (i.e., byte offsets) for storing two vectors:
+	* var xptr = 0;
+	*
+	* // Write vector values to module memory:
+	* mod.write( xptr, oneTo( N, dtype ) );
+	*
+	* // Perform computation:
+	* var out = mod.main( N, xptr, 1 );
+	* // returns 10.0
+	*/
+	new( mem: Memory ): Module; // newable
+
+	/**
+	* Returns a new WebAssembly module wrapper instance which uses the provided WebAssembly memory instance as its underlying memory.
+	*
+	* @param mem - WebAssembly memory instance
+	* @returns module wrapper instance
+	*
+	* @example
+	* var Memory = require( '@stdlib/wasm/memory' );
+	* var oneTo = require( '@stdlib/array/one-to' );
+	*
+	* // Create a new memory instance with an initial size of 10 pages (640KiB) and a maximum size of 100 pages (6.4MiB):
+	* var mem = new Memory({
+	*     'initial': 10,
+	*     'maximum': 100
+	* });
+	*
+	* // Create a BLAS routine:
+	* var mod = scasum.Module( mem );
+	* // returns <Module>
+	*
+	* // Initialize the routine:
+	* mod.initializeSync();
+	*
+	* // Define a vector data type:
+	* var dtype = 'complex64';
+	*
+	* // Specify a vector length:
+	* var N = 4;
+	*
+	* // Define pointer (i.e., byte offsets) for storing two vectors:
+	* var xptr = 0;
+	*
+	* // Write vector values to module memory:
+	* mod.write( xptr, oneTo( N, dtype ) );
+	*
+	* // Perform computation:
+	* var out = mod.main( N, xptr, 1 );
+	* // returns 10.0
+	*/
+	( mem: Memory ): Module; // callable
+}
+
+/**
+* Interface describing a `scasum` WebAssembly module.
+*/
+interface Module extends ModuleWrapper {
+	/**
+	* Computes the sum of the absolute values of the real and imaginary components of a single-precision complex floating-point vector.
+	*
+	* @param N - number of indexed elements
+	* @param xptr - input array pointer (i.e., byte offset)
+	* @param strideX - `x` stride length
+	* @returns sum of absolute values
+	*
+	* @example
+	* var Memory = require( '@stdlib/wasm/memory' );
+	* var oneTo = require( '@stdlib/array/one-to' );
+	*
+	* // Create a new memory instance with an initial size of 10 pages (640KiB) and a maximum size of 100 pages (6.4MiB):
+	* var mem = new Memory({
+	*     'initial': 10,
+	*     'maximum': 100
+	* });
+	*
+	* // Create a BLAS routine:
+	* var mod = new scasum.Module( mem );
+	* // returns <Module>
+	*
+	* // Initialize the routine:
+	* mod.initializeSync();
+	*
+	* // Define a vector data type:
+	* var dtype = 'complex64';
+	*
+	* // Specify a vector length:
+	* var N = 4;
+	*
+	* // Define pointer (i.e., byte offsets) for storing the input vector:
```

Comment:
```suggestion
	* // Define a pointer (i.e., byte offset) for storing one input vector:
```

This and similar changes throughout this PR.